tests/monitoring_integration_test.py

```diff
@@ -41,6 +46,35 @@ def getProjectId():
             'GOOGLE_CLOUD_PROJECT to your Google Cloud Project Id.')
     return project_id
 
+def getZone():
+	"""Retreives the zone of the Kubernetes cluster hosted on GKE.
+	Raises:
+	MissingZoneError -- When no cluster is found.
+	Returns:
+	str -- the zone
+	"""
+	credentials = GoogleCredentials.get_application_default()
+
+	service = discovery.build('container', 'v1', credentials=credentials)
+
+	project_id = 'linnicholas-starter-1'
```

Comment:
This needs to be read from environment variable or config file

Reply:
Yeah - it's actually retrieved earlier so I removed the statement.